src/buttons/ButtonGroup.js

```diff
@@ -42,7 +43,11 @@ const ButtonGroup = props => {
   return (
     <View
       {...attributes}
-      style={[styles.container, containerStyle && containerStyle]}
+      style={[
+        !vertical && styles.container,
+        containerStyle && containerStyle,
+        vertical && styles.verticalContainer,
```

Comment:
@k3ithl1m Always put `containerStyle` on the last line, the user should be able to override any default style.

Moreover, `styles.container` and `styles.verticalContainer` are almost the same.
In `styles.verticalContainer`, you should just override what changes in the vertical case (as far as I can see in the `styles` object in added, the `flexDirection` and the `height`)